Energy Saver/Services/TableService.cs

```diff
@@ -4,27 +4,27 @@ namespace Energy_Saver.Services
 {
     public class TableService : ITableService
```

Comment:
Weird way of deleting classes

Reply:
I wasn't sure if we would still need this so i only commented it out, we can fully delete it in develpment